app/policies/follow_policy.rb

```diff
@@ -7,6 +7,10 @@ def update?
     user_is_follower?
   end
 
+  def destroy?
+    !user_is_banned?
```

Comment:
Should we also check that the requesting user is the right one? See `user_is_follower?`